src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -327,6 +327,9 @@ StackSamplePerfEvent ConsumeStackSamplePerfEvent(PerfEventRingBuffer* ring_buffe
   perf_event_sample_id_tid_time_streamid_cpu sample_id;
   ring_buffer->ReadValueAtOffset(&sample_id, offsetof(perf_event_stack_sample_fixed, sample_id));
 
+  constexpr uint64_t kTotalNumOfRegisters =
```

Comment:
Final nit comment:
The constant should be as close as possible to its usage (so right before the ReadRawAtOffset call)

Reply:
But we already use it in initializing the event. It is defined right before it's used.

Mainly: `.regs = make_unique_for_overwrite<uint64_t[]>(kTotalNumOfRegisters),`